src/common/clickhouse.ts

```diff
@@ -5,11 +5,19 @@ let client: ReturnType<typeof createClient> | null = null;
 export const getClickHouseClient = () => {
   if (!client) {
     client = createClient({
-      url: process.env.CLICKHOUSE_URL,
+      url: 'http://localhost:18123',
```

Comment:
Don't we want this to be env based still?
Probably you added it for testing locally

Reply:
ah yeah, slipped through